### DISCUSSION TITLE
Profile Scene 작동을 위한 프로젝트 설정

### DISCUSSION THREAD

**Discussion by hgsanguk** [SeoulTechTCPGame/block-it#72] [Announcements]
## 외부 라이브러리 설명
#70 에 추가된 `Assets/Scenes/Profile/GetUserInfo.cs`는 다음 라이브러리를 포함합니다.
* `Newtonsoft.Json`: json 파일로부터 SQL Server의 로그인 정보를 불러오기 위해 사용한 라이브러리입니다.
* `System.Data.SqlClient`: SQL Server 연결과 데이터베이스 작업을 위해 필요한 라이브러리입니다. 

해당 라이브러리를 포함해 Push하였으므로 별도로 라이브러리를 설치할 필요는 없습니다. 라이브러리를 활성화하기 위해서 `File > Build Settings > Player Settings > Api Cimpatibility Level`을 `.NET Framwork`로 바꿔주시기 바랍니다.

## 서버 정보 작성
메인 화면에서 Profile Scene으로 전환되면, `Assets/Scenes/Profile/GetUserInfo.cs`가 `Assets/Resources/Json/ConnectionInfo.json`에서 SQL Server 로그인 정보를 불러옵니다. 로그인 정보이므로 해당 파일은 `.gitignore`에 추가하였고, `Assets/Resources/Json/ConnectionInfo_Template.json`의 양식과 별도로 메신저에 올릴 서버의 정보를 참고하여 `ConnectionInfo.json`을 작성해주시기 바랍니다.

문의 사항은 Comment를 달아주시기 바랍니다.

**Comment by seong0929** [comment]
`File > Build Settings > Player Settings > Player > Other Settings > Api Cimpatibility Level`
